apps/admin-ui/src/component/reservations/requested/RequestedReservation.tsx

```diff
@@ -233,6 +235,8 @@ const ReservationSummary = ({
           v: reservation?.denyReason?.reasonFi || "-",
         }
       : undefined;
+  // console.log('cancel details: ', reservation.cancelDetails);
+  console.log("handling details: ", reservation.handlingDetails);
```

Comment:
Is this log left intentionally? If so, it should be marked as such for the linter. ;)